tests/pages/api/definePassengerType.test.ts

```diff
@@ -493,6 +494,56 @@ describe('definePassengerType', () => {
         });
         await definePassengerType(req, res);
         expect(upsertPassengerTypeSpy).not.toBeCalled();
+        expect(insertPassengerTypeSpy).not.toBeCalled();
         expect(updateSessionAttributeSpy).toBeCalledWith(req, GROUP_PASSENGER_INFO_ATTRIBUTE, expect.anything());
     });
+
+    it('should save the passenger types for groups if a name is provided', async () => {
```

Comment:
If we do the above (adding check to see if duplicate name) we might need a unit test here to check that, also.

Reply:
Done 👍